test/PointerCasts/physical_pointers_vector-swizzle.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: clspv-opt %s -o %t.ll --passes=replace-pointer-bitcast
```

Comment:
That test is passing even without your fix, so it is not testing what you expect.